community/contributing/bisecting_regressions.rst

```diff
@@ -32,20 +32,27 @@ range of commits that potentially need to be built from source and tested.
 You can find binaries of official releases, as well as alphas, betas,
 and release candidates `here <https://downloads.tuxfamily.org/godotengine/>`__.
 
-For example, if you've reported a bug against Godot 3.2, you should first try to
-reproduce the bug in Godot 3.1 (not a patch release, see below for the reason).
-If the bug doesn't occur there, try to reproduce it in Godot 3.2 *beta 1* (which
-is roughly in the middle of all test builds available). If you can't reproduce
-the bug with Godot 3.2 beta 1, then try newer betas and RC builds. If you do
-manage to reproduce the bug with Godot 3.2 beta 1, then try older alpha builds.
+If you can reproduce an issue on Godot 4.0, we recommend trying to reproduce the
+issue in the latest Godot 3.x version (if the feature exhibiting the bug is
+present in 3.x). This can be used to check whether the issue is a regression in
+4.0 or not.
+
+If the issue **is present** in 3.x, then you'll need to check whether the issue
+occurs in older 3.x versions as well.
+
+If the issue is **not present** in 3.x, then you can try older 4.0 alphas and
+betas to determine when the regression started.
 
 .. warning::
 
-    For bisecting regressions, don't use patch releases such as Godot 3.1.2.
-    Instead, use the minor version's first release like Godot 3.1. This is
-    because patch releases are built from a separate *stable branch*. This kind
-    of branch doesn't follow the rest of Godot's development, which is done in
-    the ``master`` branch.
+    Project files may be incompatible between Godot versions.
+    **Make a backup of your project** before starting the bisection process.
+
+    Going from the oldest to the newest build generally reduces the risk of the
+    project not being able to successfully open in the editor in subsequent runs
+    with different engine versions. Also, the more minimal the project, the more
+    likely it'll be able to open without compatibility issues in newer engine
+    versions.
```

Comment:
```suggestion
    Going from the oldest to the newest build generally reduces the risk of the
    project not being able to successfully open in the editor, thanks to backwards
    compatibility. Try to reduce your project to the smallest repeatable example too.
    The more minimal the project is, the more likely you'll be able to open it without
    compatibility issues in newer engine versions.
```

I think the first line is a bit wordy and rich with technical jargon. Also tried to make the transition to the second half smoother.